.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+name: release
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+
+jobs:
+  release-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update Release PR
+        uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          release-type: ruby
+          package-name: next_rails
+          version-file: "lib/next_rails/version.rb"
+          changelog-types: '[{"type":"feat","section":"Features","hidden":false}, {"type":"fix","section":"Bug Fixes","hidden":false}, {"type":"chore", "section":"Miscellaneous","hidden":false}]'
```

Comment:
we can use `feat` for a `major`, `fix` for a `minor`, and `chore` for a `patch` release, [semver](]https://semver.org)
 
 Example:
 
Given our gem is currently in the `1.2.0` version
If we do the following commit: "chore(readme): Add contributions section in the readme file" 
the following release will be `1.2.1`

if we do "fix: Fix a typo" the following release will be `1.3.1`

Note: you don't need to release the gem on each commit.

Reply:
What if we want to release multiple of each at once? ie: 3 bug fixes in a single minor release, multiple features on a Major, etc?

Also this implies we are using those google conventions for writing commits, maybe that should be an RFC first.

Reply:
Example of multiple bug fixes.

Let's say we are on version 2.3.0

Then someone opens a PR to solve a fix bug  issue 1

This will create a pr changing the version to 2.3.1 and updating the changelog like this:

```yml
# Changelog

## 2.3.1 (2022-07-20)


### Bug Fix

* Description of issue 1 ([6d26c03](https://github.com/fastruby/next_rails/commit/6d26c0368e3529b850f350e0ea42ac1e5beca4d9))

...
```

Then someone opens a PR to solve a fix bug  issue 2

This time it won't create a new pr it will add notes to the changelog like this:
```yml
# Changelog

## 2.3.1 (2022-07-21)


### Bug Fix

* Description of issue 1 ([6d26c03](https://github.com/fastruby/next_rails/commit/6d26c0368e3529b850f350e0ea42ac1e5beca4d9))
* Description of issue 2 ([6d26c03](https://github.com/fastruby/next_rails/commit/6d26c0368e3529b850f350e0ea42ac1e5beca4d9))
...
```

Then someone opens a PR to solve a fix bug  issue 3

This time it won't create a new pr it will add notes to the changelog like this:
```yml
# Changelog

## 2.3.1 (2022-07-22)


### Bug Fix

* Description of issue 1 ([6d26c03](https://github.com/fastruby/next_rails/commit/6d26c0368e3529b850f350e0ea42ac1e5beca4d9))
* Description of issue 2 ([6d26c03](https://github.com/fastruby/next_rails/commit/6d26c0368e3529b850f350e0ea42ac1e5beca4d9))
* Description of issue 3 ([6d26c03](https://github.com/fastruby/next_rails/commit/6d26c0368e3529b850f350e0ea42ac1e5beca4d9))
...
```

And this won't be merged until you decide to do it manually, once, it is merged it will release the gem in RubyGems and GitHub automatically.

Does that make sense?

Reply:
@JuanVqz Let's try to implement this release workflow in `ombu_labs-auth` first. I'm not certain how this would work in practice, so `ombu_labs-auth` is small enough to learn in there instead of here.